shared/common-adapters/avatar.shared.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+import type {Props} from './avatar'
+
+export function createUrl (props: Props): ?string {
```

Comment:
🐑 Make this type be `{url: ?string} | {username: ?string}` then `Props` is just a subtype of that, and it is more general